sdk/messaging/azservicebus/example_receiver_test.go

```diff
@@ -86,3 +87,69 @@ func ExampleReceiver_ReceiveMessages() {
 		fmt.Printf("Received and completed the message\n")
 	}
 }
+
+func ExampleReceiver_DeadLetterMessage() {
+	// Create a sender client
+	sender, err := client.NewSender("myqueue", nil)
+	if err != nil {
+		panic(err)
+	}
+	defer sender.Close(context.TODO())
+	// Send a message to a queue
+	sbMessage := &azservicebus.Message{
+		Body: []byte("body of message"),
+	}
+	err = sender.SendMessage(context.TODO(), sbMessage, nil)
+	if err != nil {
+		panic(err)
+	}
+	// Create a receiver
+	receiver, err := client.NewReceiverForQueue("myqueue", nil)
+	if err != nil {
+		panic(err)
+	}
+	defer receiver.Close(context.TODO())
+	// Get the message from a queue
+	messages, err := receiver.ReceiveMessages(context.TODO(), 1, nil)
+	if err != nil {
+		panic(err)
+	}
+	// Send a message to the dead letter queue
+	for _, message := range messages {
+		deadLetterOptions := &azservicebus.DeadLetterOptions{
+			ErrorDescription: to.Ptr("exampleErrorDescription"),
+			Reason:           to.Ptr("exampleReason"),
+		}
+		err := receiver.DeadLetterMessage(context.TODO(), message, deadLetterOptions)
+		if err != nil {
+			panic(err)
+		}
+	}
+}
+
+func ExampleReceiver_GetDeadLetterMessage() {
+	// Create a dead letter reciever
+	deadLetterReceiver, err := client.NewReceiverForQueue(
+		"myqueue",
+		&azservicebus.ReceiverOptions{
+			SubQueue: azservicebus.SubQueueDeadLetter,
+		},
+	)
+	if err != nil {
+		panic(err)
+	}
+	defer receiver.Close(context.TODO())
```

Comment:
Where is `receiver` defined?

Reply:
Meant to be deadLetterReceiver*, thank you for calling that out. :)